cmd/index-buildpacks/main.go

```diff
@@ -168,6 +178,28 @@ func FetchBuildpackConfig(e Entry, imageFn ImageFunction) (Metadata, error) {
 	return m, nil
 }
 
+func FindEntry(db *sql.DB, e Entry) (bool, error) {
+	query := `
+		select exists(
+			select name
+			from buildpacks
+			where buildpacks.namespace = $1
```

Comment:
I'm not super-familiar with how indexes work for Postgres, so I could be completely mis-remembering/making this up, but does the where column order need to match the column order in the index for it to work? Or does that only affect when using eg `GROUP BY`?

ie: The existing index has the "name" column first:
https://github.com/buildpacks/registry-api/blob/80f586fee0b869a6d1c59fc4ba61fc1522926a2e/db/schema.rb#L34C14-L34C44

Reply:
My experience tells me that postgres will figure it out, but I don't mind reordering just in case it helps in some small bit.